1-JavaScript/Week3/assignment/ex2-mondaysWorth.test.js

```diff
@@ -31,15 +31,16 @@ const mondayTasks = [
 
 const hourlyRate = 25;
 
-function computeEarnings(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function computeEarnings(tasks, rate) {
+  const totalEarnings=  tasks
+      .map(task => (task.duration/60) * rate).reduce((acc, curr) => acc + curr, 0)     
+          return `€${totalEarnings}`;
```

Comment:
Can you make sure the indentation is done properly and the code is formatted well?
That way the code is more easily readable 🙏🏽 
```suggestion
  const totalEarnings = tasks
    .map(task => (task.duration/60) * rate)
    .reduce((acc, curr) => acc + curr, 0)     
  return `€${totalEarnings}`;
```